test/util/helpers.test.js

```diff
@@ -15,12 +15,11 @@ test('createGiftCardRequest', () => {
     expect(result.value.amount).toBe(20)
 })
 
-test('getEndpoint', () => {
-    const endpoint = 'EU'
-    const environment = 'production'
-    const result = getEndpoint(endpoint, endpoints, environment)
-    expect(result.location).toBe('EU')
-    expect(result.environment).toBe('production')
-    expect(result.host).toBe('agcod-v2-eu.amazon.com')
-    expect(result.region).toBe('eu-west-1')
+endpoints.forEach((endpoint) => {
```

Comment:
This is super nice. Can you separate this from the 'createGiftCardRequest' test? 

Just wrap it in something like describe('getEndpoint', () => { //code }

then cut 'getEndpoint' out of the test name